loopd/config.go

```diff
@@ -235,6 +251,30 @@ func Validate(cfg *Config) error {
 		return err
 	}
 
+	// Make sure only one of the macaroon options is used.
+	switch {
+	case cfg.Lnd.MacaroonPath != "" && cfg.Lnd.MacaroonDir != "":
+		return fmt.Errorf("use --lnd.macaroonpath only")
```

Comment:
nit: Shouldn't this be either `macaroondir` or `macaroonpath`?

Reply:
Yes. But since the `macaroondir` is deprecated, this error is a bit clearer what the action should be. And it's the exact same commit that I added to Pool, so at least the error messages will be the same :)